src/library_pthread.js

```diff
@@ -163,11 +163,8 @@ var LibraryPThread = {
 #endif
 
     runExitHandlers: function() {
```

Comment:
please add a comment here that this is run on exit on each pthread, and on the main thread (I was confused before reading enough code to realize that - I hope I understood correctly?)

Reply:
Actually, how about renaming `exitHandlers` to `threadExitHandlers` as IIUC this is just for pthreads, and *not* the main thread?

Reply:
That works, although that will be a bit redundant. These were named `PThread.exitHandlers` before, so having `PThread.threadExitHandlers` is duplicating the `thread` name. Perhaps that is worth it though.